README.md

```diff
@@ -168,11 +173,11 @@ EOF
 code .env
 ```
 
-* Ensure you replace the variable contents based on values output by the IoT Hub creation script.
-* Build the .NET app: `dotnet build`.
-* Run the .NET client app: `dotnet run`.
-* Review the upload is successful.
-* Switch to the terminal running the server app and note the notification has arrived.
+3. Ensure you replace the variable contents based on values output by the IoT Hub creation script.
+4. Build the .NET application: `dotnet build`.
+5. Run the .NET client application: `dotnet run`.
+6. Review the upload is successful.
+7. Switch to the terminal running the server application and note the notification has arrived.
```

Comment:
~~~suggestion
   Ensure you replace the variable contents based on values output by the IoT Hub creation script.

1. Build and run the .NET application.

   ```bash
   dotnet build
   dotnet run
   ```

6. Review the upload is successful.

   Switch to the terminal running the server application and note the notification has arrived.
~~~